atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/DdlConfig.java

```diff
@@ -61,6 +63,16 @@ public int mutationBatchSizeBytes() {
         return 2 * 1024 * 1024;
     }
 
+    @Value.Default
+    public int parallelDdlOperationConcurrency() {
+        return 16;
```

Comment:
Looking deeper, I realize that only one kind of operations will be performed in parallel by the executor, so this isnt a concern anymore.